langchain/vectorstores/vectara.py

```diff
@@ -0,0 +1,246 @@
+"""Wrapper around Vectara vector database."""
+from __future__ import annotations
+
+from typing import Any, List, Optional, Iterable, Tuple, Type
+import json
+import logging
+from hashlib import md5
+import requests
+
+from langchain.docstore.document import Document
+from langchain.embeddings.base import Embeddings
+from langchain.vectorstores.base import VectorStore, VectorStoreRetriever
+
+"""Implementation of Vector Store using Vectara (https://vectara.com)"""
```

Comment:
docstring a bit out of place (if here should be comment)

Reply:
> the two notebooks in chains/index_examples are a bit out place - we can't really have that for all integrations. if you want to keep around, i would move into the integrations directory and link to from the integrations page

Moved the notebooks to integrations. Thanks!

Reply:
> docstring a bit out of place (if here should be comment)

Yep. fixed!